### DISCUSSION TITLE
HUGE debug log generated by latest Integration

### DISCUSSION THREAD

**Discussion by CV-Private** [Bepacom-Raalte/bepacom-HA-Addons#60] [General]
The latest Integration is generating a HUGE amount of Debug data. I started it yesterday, and the log was as big as 24 GB today, eating up all available disk space of my HA machine to the point where HA didn't even start up any longer. I had do add more disk space to the VM to resolve the issue. 
The log file should be limited to a reasonable size.

**Comment by Bepacom-Raalte** [comment]
Do you have debug logging enabled? 
I recommend turning it off once it's running and only turn it on when encountering a problem.

**Comment by CV-Private** [comment]
It is normally off. I just turned it on out of curiosity, to see what new messages you have added. I didn't expect that it generates so much data. Anyway - I turned it off now.
